src/web/components/PortalHeader/PortalHeader.spec.tsx

```diff
@@ -17,4 +43,35 @@ describe('Portal Header tests', () => {
     const button = screen.getByRole('button');
     expect(button).toHaveTextContent('Not logged in');
   });
+
+  describe('Audit Trail in Portal Header tests based on role', () => {
+    let participantContextValue: ParticipantWithSetter;
+    let userContextValue: UserContextWithSetter;
+    let mockParticipant: ParticipantDTO;
+    let mockUser: UserWithIsUid2Support;
+
+    beforeAll(() => {
+      mockParticipant = createMockParticipant();
+      participantContextValue = createParticipantContextValue(mockParticipant);
+      mockUser = createMockUser([mockParticipant] as Participant[]);
+      userContextValue = createUserContextValue(mockUser);
+    });
+
+    test('when user has uid2support role', async () => {
+      mockUser.isUid2Support = true;
+      renderPortalHeaderWithContext(userContextValue, participantContextValue);
+      const dropdownMenuButton = screen.getByRole('button');
+      await userEvent.click(dropdownMenuButton);
```

Comment:
while calling methods on `userEvent` itself is currently supported, the [recommendation ](https://testing-library.com/docs/user-event/intro/)(and what we usually do) is to use:
`const user = userEvent.setup();` then `user.click(dropdownMenuButton);`,

Reply:
Done